tools/codegen/gen_lazy_tensor.py

```diff
@@ -201,12 +204,13 @@ def gen_key(func: FunctionSchema) -> Tuple[str, str]:
 
     # Generate nativefunction declarations
     gen_dispatchkey_nativefunc_headers(fm, class_name, cpp_namespace, backend_indices,
```

Comment:
could you add a comment explaining why eager_registration=False for lazy TS backend?